README.md

```diff
@@ -84,26 +84,42 @@ Please note — packaged into an asar archive [by default](https://github.com/el
 * `-amd64.deb` and `-i386.deb`: Linux Debian package. Please note — by default the most effective [xz](https://en.wikipedia.org/wiki/Xz) compression format used.
 
 You need to deploy somewhere releases/downloads server.
-Consider to use [Nuts](https://github.com/GitbookIO/nuts) (GitHub as a backend to store assets) or [Electron Release Server](https://github.com/ArekSredzki/electron-release-server).
+Consider using [Nuts](https://github.com/GitbookIO/nuts) (GitHub as a backend to store assets) or [Electron Release Server](https://github.com/ArekSredzki/electron-release-server).
 
 # Code Signing
 OS X and Windows code singing is supported.
-On a development machine set environment variable `CSC_NAME` to your identity (recommended). Or pass `--sign` parameter.
+
+## Signing OS X Development Builds
+On a development machine, set environment variable `CSC_NAME` to your identity (recommended). Or pass `--sign` parameter.
 ```
 export CSC_NAME="Developer ID Application: Your Name (code)"
 ```
 
-## Travis, AppVeyor and other CI servers
+This is only required on OS X.
+
+## Signing Production Builds on Travis, AppVeyor and other CI servers
 To sign app on build server:
 
-1. [Export](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingCertificates/MaintainingCertificates.html#//apple_ref/doc/uid/TP40012582-CH31-SW7) certificate.
- [Strong password](http://security.stackexchange.com/a/54773) must be used. Consider to not use special characters (for bash) because “*values are not escaped when your builds are executed*”.
-2. Upload `*.p12` file (e.g. on [Google Drive](http://www.syncwithtech.org/p/direct-download-link-generator.html)).
+> **NOTICE**: When exporting a code signing certificate, a [strong password](http://security.stackexchange.com/a/54773) must be used.
+> Avoid using special characters (for bash) because “*values are not escaped when your builds are executed*”.
+
+1. Get a code signing certificate
+  - OS X
```

Comment:
Oh, I bought an Apple Developer account *and* DigiCert certificate just for this because I thought I had to. I couldn't find anything on the internet that said otherwise. We should definitely recommend cheaper alternatives.